FV3/wrapper/lib/Makefile

```diff
@@ -2,29 +2,28 @@ SHELL = /bin/sh
 
 TEMPLATES = $(shell ls ../templates)
 PROPERTIES_FILES = ../fv3gfs/wrapper/dynamics_properties.json ../fv3gfs/wrapper/physics_properties.json
+include $(MAKE_INCLUDE)
```

Comment:
Can you add a ?= on MAKE_INCLUDE pointing to the conf/configure.fv3 so that it's clear from the code what this file is? And so this makefile still runs standalone.

Reply:
> And so this makefile still runs standalone.

I wanted to avoid this to make the entry point as simple as possible, but it could be a good place to add a warning message about not doing that. I may also refactor these build statements to `FV3/makefile`.

Reply:
I removed this makefile. See [8eaf976](https://github.com/ai2cm/fv3gfs-fortran/pull/280/commits/8eaf97677ce2d796592142b6fed5afdd9fb27ce1).